script.js

```diff
@@ -1,21 +1,329 @@
+closeMobilePopupWindow.src = './Icons/closePopup.png';
+closeMobilePopupWindow.alt = 'close popup';
+
+// popupWindowContent.append(closeMobilePopupWindow);
+
+for(let i = 0; i < projects.length; i++) {
+
+    // Create popupImageBox with it content and append it to popupWindowContent
+    popupImageBoxs[i] = document.createElement('div');
+    popupImageBoxs[i].className = 'projectImageBox';
+    popupImages[i] = document.createElement('img');
+    popupImages[i].src = projects[i].image;
+    popupImages[i].alt = 'project image';
+    popupImageBoxs[i].appendChild(popupImages[i]);
+
+    //Create popup window projectTextBox 
+    popupTextBoxs[i] = document.createElement('div');
+    popupTextBoxs[i].classList.add('projectTextBox', 'flex-column');
+
+    // Create popupTitleBox with it content and append it to popup window projectTextBox
+    popupTitleBoxs[i] = document.createElement('div');
+    popupTitleBoxs[i].className = 'projectTitleBox';
+    popupTitles[i] = document.createElement('h2');
+    popupTitles[i].innerHTML = projects[i].name;
+    popupTitleBoxs[i].appendChild(popupTitles[i]);
+    popupTextBoxs[i].appendChild(popupTitleBoxs[i]);
+
+    // Create popupTechList
+    popupTechList[i] = document.createElement('ul');
+    popupTechList[i].classList.add('projectTechList', 'flex-row');
+
+    // Create projectTechListItems with their link and append them to projectTechList
+    for(let j = 0; j < projects[i].technologies.length; j++) {
+        popupTechItems[j] = document.createElement('li');
+        popupTechLinks[j] = document.createElement('a');
+        popupTechLinks[j].href = projects[i].technologies[j].url;
+        popupTechLinks[j].target = '_blank';
+        popupTechLinks[j].rel = 'noopener noreferrer';
+        popupTechLinks[j].innerHTML = projects[i].technologies[j].name;
+        popupTechItems[j].appendChild(popupTechLinks[j]);
+        popupTechList[i].appendChild(popupTechItems[j]);
+    }
+
+    // Append projectTechList to projectTextBox
+    popupTextBoxs[i].appendChild(popupTechList[i]);
+
+
+    // Mobile screen size popup window content creation Start
+    // Create projectDescription
+    projectDescriptions[i] = document.createElement('div');
+    projectDescriptions[i].innerHTML =  projects[i].description;
+    projectDescriptions[i].classList.add('projectDescription', 'flex-column');
+
+
+    // Create popupButtonsBox
+    popupButtonsBoxs[i] = document.createElement('div');
+    popupButtonsBoxs[i].classList.add('popupButtonsBox', 'flex-row');
+
+    // Create PopupLiveButton and append it to popupButtonBox
+    popupLiveButtonBoxs[i] = document.createElement('div');
+    popupLiveButtonBoxs[i].classList.add('popupLiveButtonBox', 'flex-row');
+    popupLiveLinks[i] = document.createElement('a');
+    popupLiveLinks[i].href = projects[i].liveVersion;
+    popupLiveLinks[i].target = '_blank';
+    popupLiveLinks[i].rel = 'noopener noreferrer';
+    popupLiveLinks[i].innerHTML = 'See live';
```

Comment:
- [OPTIONAL] Great job on implementing the pop functionality. :+1: :100: , I noticed that the `See Live` and `See Source` button on your pop-up does not have the button interaction styling added to them 'hover, clicked'. Please implement this so that it can stay consistence with your design and also for the optimum user experience. :smile: :smile: